tools/static_word_checks.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+from typing import Any, Callable
+
+
+class StaticCheck:
+    """Run simple checks on the entire document or specific lines."""
+
+    def __init__(self):
+        self.directory = os.path.abspath(
```

Comment:
This variable could be determined inside `run_checks`. Can we make it local to there?